### ISSUE TITLE
Doesn't build today - declaration issue

### ISSUE THREAD

**Issue by xach** [cbaggers/cepl#248]
Build log is here: http://report.quicklisp.org/2018-02-19/failure-report/cepl.html#cepl

**Comment by cbaggers** [comment]
@xach Thanks for raising thins. That is probably due to `rtg-math` and should go away when https://github.com/quicklisp/quicklisp-projects/issues/1454 takes effect.

**Comment by xach** [comment]
I'm using rtg-math from that branch already - no change in result.

**Comment by cbaggers** [comment]
damn. Thankyou I'll look into this tonight

**Comment by cbaggers** [comment]
@xach I have pushed the newer code so hopefully this has fixed this case. One lovely feature to have would be for the failure report to report a change in status. Currently it's reporting when it starts failing, but info on when it starts working again would let me tell you more reliably that it was fixed.

Regardless though the failure-reports are excellent and I love that I can watch them on a per author feed as well. Top stuff!
